docs/synthetic-tokens/glossary.md

```diff
@@ -14,6 +14,38 @@ A token sponsor is an entity who bears the financial risk of the synthetic token
 Token sponsors have short exposure to the price identifiers of the synthetic tokens they sponsor.
 
+## Parameters of a Long Short Pair (LSP) smart contract:
+
+### Financial Product Library (FPL)
+
+The financial product libraries are used to transform the value returned by the price identifier into a final settlement value. Financial product libraries can be applied to create different types of financial contracts and payout functions.
+
+Refer to [github](https://github.com/UMAprotocol/protocol/tree/master/packages/core/networks) for a list of deployed financial product libraries for each network. If your desired financial product library is not already deployed, refer [here](https://github.com/UMAprotocol/launch-emp#deploying-financial-product-libraries) for instructions on deploying and verifying your own financial product library contract.
+
+### collateralPerPair
+
+The collateralPerPair parameter determines the amount of collateral required to mint each pair of long and short tokens.
+
+- Example: If a contract uses WETH as collateral and the collateralPerPair parameter is set to 0.25 on deployment, each long and short token that is minted would require 0.25 WETH as collateral.
+
+### expiryPercentLong
+
+ExpiryPercentLong is used to determine the redemption rate between long and short tokens. ExpiryPercentLong is a number between 0 and 1, where 0 assigns all collateral to the short tokens and 1 assigns all collateral to the long tokens.
+
+## Products that use the LSP contract:
+
+### Range Tokens
+
+Range tokens enable a DAO or nascent project to use its native token as collateral to borrow funds. At maturity, if the debt is not paid, the range token holder is instead compensated with an equivalent amount of the collateral (the native token) using the settlement price of the native token to determine the number of tokens. Go [here](/range-tokens/summary) to learn more about range tokens and how to deploy your own contract.
+
+### Success Tokens
+
+Success tokens offer an alternative way for DAOs to diversify their treasury and sell tokens to investors in an incentive-aligned way. Success tokens are just two financial products wrapped into one token: a set amount of a project token which is combined with a covered call option on that token backed by a set amount of the same token. Go [here](/success-tokens/summary) to learn more about range tokens and how to deploy your own contract.
+
+### KPI Options
+
+Key Performance Indicator (KPI) Options are synthetic tokens that will pay out more rewards if a project’s KPI reaches predetermined targets before a given expiry date. Every KPI Option holder has an incentive to improve that KPI because their option will be worth more. This is intended to align individual token holder interests with the collective interests of the protocol. Go [here](/kpi-options/summary) to learn more about range tokens and how to deploy your own contract.
```

Comment:
```suggestion
Key Performance Indicator (KPI) Options are synthetic tokens that will pay out more rewards if a project’s KPI reaches predetermined targets before a given expiry date. Every KPI Option holder has an incentive to improve that KPI because their option will be worth more. This is intended to align individual token holder interests with the collective interests of the protocol. Go [here](/kpi-options/summary) to learn more about KPI options and how to create your own.
```